src/components/Forms/Signatures/style.module.less

```diff
@@ -18,4 +18,20 @@
     margin-top: 0;
     margin-bottom: 3rem;
     font-size: @fontSizeM;
+}
+
+.anonymousListDownloadInfo {
+    margin-top: 6rem;
+
+    @media (max-width: @breakPointS) {
+        margin-top: 21rem;
+    }
+}
+
+.illustrationRotated img {
```

Comment:
hey, this works, but I think it has the potential to get messy. It's hard to maintain if you have these kind of side-effects. Because when you do stuff inside the component, there is no way to know of this kind of usage and modification.

Now I get that we are using the `CTAButtonContainer` in another place where we want a different positioning...

I guess the actual problem is that maybe having the illustration be a part of the CTAButton is not really the best idea. It could be a wrapper that just generates an illustration pointing on whatever is in that wrapper. That could have a prop for different positioning needs.